provider/filestore.go

```diff
@@ -266,13 +267,30 @@ func (s3 *S3FileStore) PathWithPrefix(path string, remote bool) string {
 }
 
 type GCSFileStore struct {
+	Bucket string
+	Path   string
 	genericFileStore
 }
 
+func (gs *GCSFileStore) PathWithPrefix(path string, remote bool) string {
+	gsPrefix := "gs://"
+	noGSPrefix := path[:len(gsPrefix)] != gsPrefix
+
+	if remote && noGSPrefix {
+		gsPath := ""
```

Comment:
If this is always an empty string why even have this here?

Reply:
this is used to populate the user defined prefix path. The naming might be confusing so I changed it.